src/components/Buttons/GoBackButton/GoBackButton.tsx

```diff
@@ -1,9 +1,10 @@
 import { Link } from "react-router-dom";
 import styles from './GoBackButton.module.scss';
+import { RoutesLink } from "../../../types/routes";
 
 export const GoBackButton = () => {
   return (
-    <Link to='#' className={styles.go_back}>
+    <Link to={RoutesLink.HomePage} className={styles.go_back}>
```

Comment:
It's also can be other route of each page, so you should implement this functionality using useLocation